media/css/firefox/welcome.scss

```
@@ -360,3 +367,181 @@ $image-path: '/media/protocol/img';
         display: none;
     }
 }
+
+// hide alt hero images
+#hero-image,
+#hero-animation {
+  display: none;
+}
+
+
+.welcome-page8.variation-2-image,
+.welcome-page8.variation-3-animation {
+  .mzp-l-content {
+    padding: 24px 52px;
+  }
+
+  // hide default hero image
+  .mzp-c-hero.variation-default {
+    display: none;
+  }
+
+  .mzp-c-hero-title {
+    max-width: $content-md;
+    margin: 0 auto;
+  }
+
+  .image-container {
+    margin: 35px 0 70px 0;
+
+    .image-wrapper {
+      display: inline-block;
+      vertical-align: top;
+      margin: 0 50px 0 50px;
+      position: relative;
+
+      p {
+        font-size: 17px;
```

Nit: use `font-size()` mixin for `rem` support.
```suggestion
        @include font-size(17px);
```